src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualHub.md

```diff
@@ -0,0 +1,140 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
```

Comment:
I'll check the help files once parameter sets have been properly implemented and example output has been given.